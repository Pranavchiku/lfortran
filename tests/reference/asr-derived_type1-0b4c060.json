{
    "basename": "asr-derived_type1-0b4c060",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/derived_type1.f90",
    "infile_hash": "ed39e2004e29457f0f83e6a1badf6ecbee98749ca624594fc0160e52",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-derived_type1-0b4c060.stdout",
    "stdout_hash": "3e46a5c785a19758c5576a524b628408b3a35b41d6d541ec34534cf1",
    "stderr": "asr-derived_type1-0b4c060.stderr",
    "stderr_hash": "63762e25d69bb470d2363165aa95d4a7ee27d415978763a548df0abc",
    "returncode": 0
}