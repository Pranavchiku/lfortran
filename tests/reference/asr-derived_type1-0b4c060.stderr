warning: Implied save feature
 --> tests/derived_type1.f90:4:5
  |
4 |     integer, public, parameter :: tfc = selected_char_kind('DEFAULT')
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/derived_type1.f90:7:9
  |
7 |         character(len=1, kind=tfc) :: newline = achar(10, kind=tfc)
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/derived_type1.f90:10:5
   |
10 |     type(enum_escape), public, parameter :: toml_escape = enum_escape()
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/derived_type1.f90:11:5 - 12:33
   |
11 |        character(kind=tfc, len=*), public, parameter :: nl = &
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
   |
12 |            &' '//toml_escape%newline
   | ...^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
